relion/_parser/ctffind.py

```diff
@@ -127,17 +99,6 @@ def parse_star_file(self, loop_name, star_doc, block_number):
         values_list = list(values)
         if not values_list:
             print("Warning - no values found for", loop_name)
-        return values_list
```

Comment:
This function no longer has a return value. Deleted by mistake?

Reply:
Ahh.... yes, mistake